release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3090,3 +3090,36 @@ link:https://access.redhat.com/solutions/6145452[{product-title} 4.6.36 containe
+
+* Previously, CRI-O logs did not contain information about the source where images were pulled from. Consequently, CRI-O failed to tell if the image was pulled from the registry mirror. This update adds level log information to CRI-O about the pull source of the image. As a result, the pull source is now whoen in the information level of the log. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1976293[*BZ#1976293*])
+
+* Previously, the authorization header created during image mirroring could exceed the header size limit for some registries. This would cause an error during the mirroring operation. Now, the `--skip-multiple-scopes` option is set to `true` for the `oc adm catalog mirror` command, to help prevent the authorization header from exceeding the header size limits. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1946839[*BZ#1946839*])
+
+* Before this update, the Pipeline `ServiceAccount` did not use secrets created during the `git import` flow for private Git repositories, which caused these Pipelines to fail. This update fixes the problem by adding annotations to the secrets and to the Pipeline `ServiceAccount`. Pipelines for private Git repositories now run correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970470[*BZ#1970470*])
+
+* Previously, not all options from config were copied when switching projects. Consequently, when using switching projects using `Exec` to authenticate in kubeconfig, information was lost. This update copies all the necessary information when switching projects using `oc projects` and `Exec` to authenticate. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1973613[*BZ#1973613*])
+
+* Previously, when a second internal IP address was added to one or more control plane nodes, the etcd Operator degraded due to detecting the IP address change as a potential membership change. Consequently, it did not regenerate etcd serving certificates for the node. With this update, the etcd Operator differentiates between an IP address change for new and existing nodes. As a result, the etcd Operator regenerates serving certificates for changes to an existing node, and adding a new IP address no longer causes the etcd Operator to degrade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1965535[*BZ#1965535*])
+
+* Previously, the topology URLs created for deployments using the Bitbucket repository in the {product-title} web console did not work if they included a branch name that contained a slash (`/` or `\`) character. This was due to an issue with the Bitbucket API (link:https://jira.atlassian.com/browse/BCLOUD-9969[*BCLOUD-9969*]). The current release mitigates this issue. If a branch name contains a slash, the topology URLs point to the default branch page for the repository. This issue will be fixed in a future release of {product-title}. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972694[*BZ#1972694*])
```

Comment:
Based on the "Individual punctuation marks or special characters" section, this would be better:

```suggestion
* Previously, the topology URLs created for deployments using the Bitbucket repository in the {product-title} web console did not work if they included a branch name that contained a backslash (\) or forward slash (/). This was due to an issue with the Bitbucket API (link:https://jira.atlassian.com/browse/BCLOUD-9969[*BCLOUD-9969*]). The current release mitigates this issue. If a branch name contains either a backslash or forward slash, the topology URLs point to the default branch page for the repository. This issue will be fixed in a future release of {product-title}. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972694[*BZ#1972694*])
```